esmvaltool/diag_scripts/perfmetrics_main.ncl

```diff
@@ -200,17 +211,24 @@ begin
         error_msg("f", diag_script, "", "no reference model is specified")
     end if
 
-    ;; Determine reference (and alternative) model
-    ref_ind = get_ref_model_idx(models, var_attr_ref)
-    if (dimsizes(ref_ind).gt.1) then
-        alt_ind = ref_ind(1)
-        ref_ind := ref_ind(0)
+    ;; Set index of the reference (and alternative) model
```

Comment:
The name of the reference and alternative model are now available as `variable_info@reference_model` and `variable_info@alternative_model`. Maybe it would be better to use those to get the model indices instead of `var_attr_ref`, as the relation with the definition in the namelist is more clear that way. This could also be addressed later as part of issue #182.

Reply:
Good point!
As you suggested, I would address this together with the other issues in [#182](https://github.com/ESMValGroup/ESMValTool/issues/182) in a separate branch.
I added a reminder in the code.